sites/headers-includes/header.html

```diff
@@ -1,16 +1,33 @@
 <header>
 	<div id="wb-bnr" class="container">
 		<div class="row">
-			{% if page.altLangPage %}{% include language/inc-languagetoggle.html %}{% endif %}
+			{% if page.altLangPage %}
+				{% include language/inc-languagetoggle.html %}
+			{% endif %}
 			{% include headers-includes/brand.html %}
-			{% unless page.nositesearch %}{% include headers-includes/sitesearch.html %}{% endunless %}
+			{% unless page.nositesearch %}
+				{% include headers-includes/sitesearch.html %}
+			{% endunless %}
 		</div>
 	</div>
-{%- unless page.nomenu -%}
-	{% include gcweb-menu/inc-sitemenu.html %}
-{%- endunless -%}
-{% include breadcrumbs/inc-breadcrumbs.html %}
-{%- if page.signing -%}
-	{% include headers-includes/sign-on-off.html %}
-{%- endif -%}
+	<hr>
+	<div class="container">
+		<div class="row">
+			{%- unless page.nomenu -%}
```

Comment:
What happen if there is "nomenu" but there is "auth" ? Is the "auth" require to have a menu?
An additional 1 working example would be required to illustrate the situation where there is no menu but an authentication component.

Reply:
@duboisp Any pattern in particular I should use: Contextual or Signed-Off? Where does the nomenu with authentication pattern example should reside...under gcweb-menu?

Reply:
One or the other, we can cross-link the demo via the index.json-ld file of both site component